plugin/src/withSpotlight.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  ConfigPlugin,
+  createRunOncePlugin,
+  WarningAggregator,
+  withAppDelegate,
+} from "@expo/config-plugins";
+
+const pkg = require("react-native-spotlight-search/package.json");
+
+const SPOTLIGHT_IMPORT = `#import "RCTSpotlightSearch.h"
+`;
+const SPOTLIGHT_ACTIVITY = `- (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull NSUserActivity *)userActivity restorationHandler:(nonnull void (^)(NSArray<id<UIUserActivityRestoring>> * _Nullable))restorationHandler {`;
+
+const SPOTLIGHT_ACTIVITY_ADD = `- (BOOL)application:(UIApplication *)application continueUserActivity:(nonnull NSUserActivity *)userActivity restorationHandler:(nonnull void (^)(NSArray<id<UIUserActivityRestoring>> * _Nullable))restorationHandler {
```

Comment:
Moving forwards, if we change the native code for non-expo projects, do we need to keep these in sync?

Reply:
I don't think this piece of code will change. This is the method that is automatically generated when creating a react-native app. But yes, If native code changes then there are chances that we'll need to update this as well.